### PR TITLE
1.1/144 reset password functionality

### PR DESCRIPTION
**Pull request by charlesstrange2**
Fixed up the ARC Admin side of Password resets. 
API User resets to come but that looks like another chunk of work.
The vendor template is also ugly and could do with independent prettification.

### CODE REVIEW

File: app/Notifications/AdminPasswordResetNotification.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    /**
+     * Get the array representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function toArray($notifiable)
+    {
+        return [
+            //
```
Comment (katjam) [inline app/Notifications/AdminPasswordResetNotification.php]: Do we want to us this?

File: app/Notifications/AdminPasswordResetNotification.php [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+            ->subject('Password Reset Request Notification')
+            ->greeting('Hello!')
+            ->line('You are receiving this email because we received a password reset request for your account on the Rosie admin service.')
```
Comment (katjam) [inline app/Notifications/AdminPasswordResetNotification.php]: xXx ?

File: app/Notifications/AdminPasswordResetNotification.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+            ->subject('Password Reset Request Notification')
+            ->greeting('Hello!')
```
Comment (katjam) [inline app/Notifications/AdminPasswordResetNotification.php]: User's name?

File: app/Notifications/AdminPasswordResetNotification.php (hunk 1 above)
Comment (charlesstrange2) [inline app/Notifications/AdminPasswordResetNotification.php]: It seems to be a default method, and is present (and unspecified) in other notifications created by `php artisan make:notification`

File: app/Notifications/AdminPasswordResetNotification.php (hunk 2 above)
Comment (charlesstrange2) [inline app/Notifications/AdminPasswordResetNotification.php]: Yup, will add

File: app/Notifications/AdminPasswordResetNotification.php (hunk 3 above)
Comment (charlesstrange2) [inline app/Notifications/AdminPasswordResetNotification.php]: Added

**Pull request merged**